templates/publisher/market.html

```diff
@@ -187,15 +187,6 @@
             </div>
           </div>
 
-          <div class="row">
-            <div class="col-2">
-              <label>License: </label>
-            </div>
-            <div class="col-8">
-              {{ license }}
-            </div>
-          </div>
-
           <div class="row">
```

Comment:
You should also remove one of the rows with `<hr/>` from above or below license.

Reply:
Thanks @bartaz I thought I had removed it! Fixed.